src/connectors/status/status-db.connector.js

```diff
@@ -144,4 +171,15 @@ const updateStoredStatus = async (statusName, newStatus) => {
   }
 };
 
-module.exports = { getStoredStatus, updateStoredStatus, getEmailDistribution };
+const clearMongoConnection = () => {
+  client = undefined;
```

Comment:
you should conditionally close the connection here as per above, although this code is only called in tests, right? I assume it's necessary?

Reply:
Yes, only for testing purposes. Similar pattern exists in other modules.